export const doThrow = (error: unknown) => {
  throw error;
};
